python/ray/tests/chaos/fine_tuning_like.py

```diff
@@ -0,0 +1,140 @@
+"""
+Generator data that uses plasma store (XGB, I am thinking 1GB for the first try)
+Process data (1~2 seconds, just random processing)
+Send data to Y actors (24 actors in the first try)
+Actors are sleeping for 30 seconds
+Repeat
+"""
+
+import argparse
+import ray
+import numpy as np
+import time
+import threading
+import asyncio
+from contextlib import contextmanager
+
+
+@ray.remote(num_cpus=1)
+def read_task():
+    # Waste resources
+    for _ in range(10):
+        np.random.rand(5 * 1024 * 1024)
+
+    # 8 MB * 4 == 32MB
+    return [np.random.rand(1024 * 1024) for _ in range(4)]
+
+
+@ray.remote(num_cpus=1)
+def preproc(refs):
+    n = np.random.rand(1024 * 1024)
+    arrays = ray.get(refs)
+    for _ in range(10):
+        for arr in arrays:
+            n += arr[0]
+    return n
+
+
+@ray.remote(num_cpus=0)
+class Coord:
+    def __init__(self, actors_in_group):
+        self.actors_in_group = actors_in_group
+
+    async def run(self, preproc_refs):
+        while True:
+            await asyncio.sleep(1)
+            refs = []
+            for a in self.actors_in_group:
+                refs.append(a.ping.remote())
+            await asyncio.wait(refs)
+            done = True
+            for ref in refs:
+                finished = await ref
+                if not finished:
+                    done = False
+            if done:
+                break
+
+
+@ray.remote(num_cpus=0)
+class FakeFineTuninig:
```

Comment:
I wonder if it's possible we use Ray Tune with a fake tune load? something like (example from https://docs.ray.io/en/latest/tune/index.html):

```
from ray import train, tune


def objective(config):  # ①
    score = config["a"] ** 2 + config["b"]
    return {"score": score}


search_space = {  # ②
    "a": tune.grid_search([0.001, 0.01, 0.1, 1.0]),
    "b": tune.choice([1, 2, 3]),
}

tuner = tune.Tuner(objective, param_space=search_space)  # ③

results = tuner.fit()
print(results.get_best_result(metric="score", mode="min").config)
```

Reply:
actually it is not a tune test. It is "fine tuninig" (which means you run training workload on top of existing weights). Re; using library directly, I prefer to just minimze the lib dependencies